hack/collector-default

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+###############################################################################
+# host information
+###############################################################################
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
```

Comment:
Since the `collector-default` will always execute. We should check other collectors to avoid duplicate collections.